### PR TITLE
refactor: bundle maps ourselves

### PR DESCRIPTION
**Pull request by tpluscode**
I propose to replace the map bundle with our own where we import what's necessary.

### CODE REVIEW

File: packages/yasgui/package.json [hunk 1]
```diff
@@ -17,20 +17,22 @@
...
+    "dist",
     "index.js"
   ],
   "scripts": {
+    "prepare": "esbuild --minify --bundle plugins/map.js --outdir=dist --splitting --format=esm",
     "test": "c8 --all --reporter=lcovonly --reporter=text mocha"
   },
   "dependencies": {
     "@fastify/static": "^7.0.4",
-    "@openlayers-elements/bundle": "^0.1.0",
```
Comment (tpluscode) [inline packages/yasgui/package.json:28]: Replaced with `@openlayers-elements/maps` as dev dependency

File: packages/yasgui/package.json [hunk 2]
```diff
@@ -17,20 +17,22 @@
   "files": [
     "test",
     "views",
-    "plugins",
+    "dist",
     "index.js"
   ],
   "scripts": {
+    "prepare": "esbuild --minify --bundle plugins/map.js --outdir=dist --splitting --format=esm",
```
Comment (tpluscode) [inline packages/yasgui/package.json:24]: By splitting and dynamic importing the map components we reduce the initial load

**Comment by tpluscode** [comment]
https://github.com/user-attachments/assets/9b98875c-8940-483e-ab43-717a3c8133ce

**Review by ludovicm67 — APPROVED** [review]
@tpluscode Thanks a lot for your suggestions.

I also used that PR to add a `mapKind` option and improved the way the map plugin gets rendered.

**Pull request merged**